pkg/autoupdate/autoupdate.go

```diff
@@ -282,6 +332,21 @@ func newerImageAvailable(runtime *libpod.Runtime, img *image.Image, origName str
 	return img.Digest().String() != remoteDigest.String(), nil
 }
 
+// newerLocalImageAvailable returns true if the container and local image are different
+func newerLocalImageAvailable(ctr *libpod.Container, img *image.Image) (bool, error) {
+
+	//ctr_digest := ctr.Digest().String()
+	//img_digest := img.Digest().String()
+
+	//return ctr_digest == img_digest, nil
+
+	img_created := img.Created()
+	ctr_created := ctr.CreatedTime()
```

Comment:
I think we should continue comparing digests rather than using time stamps.  The latter is prone to errors.